examples/0_Simple/vectorSqrt/device.gpr

```diff
@@ -11,9 +11,16 @@ project Device is
 
    for Create_Missing_Dirs use "True";
 
+   for Library_Name use "kernel";
+   for Library_Dir use "lib";
+
+   for Archive_Builder use ("cuda-gcc", "-v");
+   for Archive_Indexer use ("");
+   for Archive_Suffix use ".fatbin.o";
+
    package Compiler is
-      for Switches ("ada") use ("-O2", "-gnatn", "-nostdlib");
+      for Switches ("ada") use ("-gnatX", "-O2", "-gnatp", "-gnatn", "-v");
```

Comment:
it seems that the flags changed?
rm -nostdlib
add -gnatp, -gnatX, and -v

Are they all expected?

Reply:
Right, `-nostdlib` is actually harmful (we want to build against the stdlib, which will contain necessary symbols) and was a leftover from previous experiments.

`-O2`, `-gnatp` and `-gnatn` are perf flags.

`-v` helps with debugging issues, should they arise.